packages/grid/x-data-grid-pro/src/tests/cellEditing.DataGridPro.test.tsx

```diff
@@ -7,6 +7,7 @@ import {
   DataGridPro,
 } from '@mui/x-data-grid-pro';
 import Portal from '@mui/base/Portal';
+// @ts-ignore
```

Comment:
It would be nice to type them at some point but yes we can ignore for now

Could you add an explicit description
That way we will be able to easily remove all of them with a search and replace once it's typed

```suggestion
// @ts-ignore Remove once the test utils are typed
```